static/app/views/starfish/views/spans/spansView.tsx

```diff
@@ -66,20 +42,20 @@ export default function SpansView(props: Props) {
         {[ModuleName.ALL, ModuleName.OTHER].includes(moduleName) && (
           <SpanOperationSelector
             moduleName={moduleName}
-            value={appliedFilters[SPAN_OP] || ''}
+            value={moduleFilters[SPAN_OP] || ''}
```

Comment:
l: `Query` type in `useModuleFilters` says that a string will always be returned. Do we still need the `|| ''` check?

Reply:
That type is lying, it does _not_ always return a string. Updated the type